cookie-banner-rules-list.json

```diff
@@ -1639,7 +1587,18 @@
         "orange.fr",
         "meteofrance.com",
         "subito.it",
-        "hasznaltauto.hu",
+        "hasznaltauto.hu"
+      ]
+    },
+    {
+      "click": {
+        "optIn": "#didomi-notice-agree-button",
+        "optOut": ".didomi-continue-without-agreeing",
+        "presence": "div#didomi-notice"
+      },
+      "cookies": {},
+      "id": "688d29a8-e1c7-4d62-b3d4-53b451ff5a48",
+      "domains": [
         "orange.sk"
       ]
```

Comment:
There are already rules for didomi, please check if this rule can be merged into those.